Cargo.toml

```diff
@@ -11,6 +11,7 @@ members = [
 ]
 
 exclude = [
+    "lightning-custom-message",
```

Comment:
Trying to compile using `cargo +1.41.1 check` gives:

```
error: failed to parse manifest at `/Users/jkczyz/src/rust-lightning/lightning-custom-message/Cargo.toml`

Caused by:
  failed to parse the `edition` key

Caused by:
  supported edition values are `2015` or `2018`, but `2021` is unknown
```

But after rebasing, it looks like that's also a problem for one one of `lightning-transaction-sync`'s dependencies, as well. I guess we just can't build the entire workspace with an older rust version now?

Reply:
Ok, correction, the above command also doesn't work because `lightning-net-tokio` requires a higher version. So moving `lightning-custom-message` back into the members should be fine.

Reply:
Spoke to soon. CI is unhappy because of the edition:

```
error: failed to parse manifest at `/home/runner/work/rust-lightning/rust-lightning/lightning-custom-message/Cargo.toml`

Caused by:
  failed to parse the `edition` key

Caused by:
  supported edition values are `2015` or `2018, but `2021` is unknown
```